web/app/js/components/TabbedMetricsTable.jsx

```diff
@@ -79,14 +79,6 @@ export default class TabbedMetricsTable extends React.Component {
   constructor(props) {
     super(props);
     this.api = this.props.api;
-
-    this.state = {
-      timeseries: {},
-      rollup: this.preprocessMetrics(),
-      error: '',
-      pollingInterval: 10000,
-      pendingRequests: false
-    };
   }
```

Comment:
Just to clarify: we don't need to have a state here with `pollingInterval`, as all parent components have it and will trigger a re-render upon fetching a new interval which will trigger down a new and updated render of `TabbedMetricsTable`, right?

Reply:
correct! we fetch data every pollingInterval in the parent component, and pass the data down as a prop to TabbedMetricTable. When the TabbedMetricsTable receives the new props, it will rerender. 
Previously, we were setting the state based on the props in the constructor - this doesn't update the state when new props are passed in, which is why the table wasn't updating.

Reply:
wonderful! Thanks for confirming / answering my question! 😻 💫